docs/examples/balltree-kdtree-performance.ipynb

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
<p>Add some command to print the size of the file GB, also add the number of nodes/faces  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Printing the <code>__repr__</code> for the <code>Grid</code> should be sufficient (i.e. just printing <code>grid</code>)  </p>